src/Microsoft.ML.DataView/DataViewSchema.cs

```diff
@@ -12,7 +12,7 @@
 namespace Microsoft.ML
 {
     /// <summary>
-    /// This class represents the <see cref="DataViewSchema"/> of an object like, for interstance, an <see cref="IDataView"/> or an <see cref="DataViewRow"/>.
+    /// This class represents the <see cref="DataViewSchema"/> of an object like, for instance, an <see cref="IDataView"/> or an <see cref="DataViewRow"/>.
```

Comment:
Do we need the "like" in this sentence?

Suggest: This class represents the &lt;see cref="DataViewSchema"/&gt; of an &lt;see cref="IDataView"/&gt; object or a &lt;see cref="DataViewRow"/&gt; object.